packages/hooks/src/useIsGrowthbookLoaded.ts

```diff
@@ -5,16 +5,22 @@ import { useRemoteConfig } from '@deriv/api';
 const useIsGrowthbookIsLoaded = () => {
     const [isGBLoaded, setIsGBLoaded] = useState(false);
     const { data } = useRemoteConfig(true);
+    const [isGBNotAvailable, setIsGBNotAvailable] = useState<boolean>(false);
 
     useEffect(() => {
         let analytics_interval: NodeJS.Timeout;
+
+        if (/^(localhost|127\.0\.0\.1|::1)(:\d+)?$/i.test(window.location.hostname)) {
+            setIsGBNotAvailable(true);
```

Comment:
usually better to use positive signals - "isGBAvaiallbe"
(our brains are way better dealing with positive logic than dealing with negative one)